### DISCUSSION TITLE
Multiple authors in blog?

### DISCUSSION THREAD

**Discussion by kaosine** [vuepress-theme-hope/vuepress-theme-hope#2250] [Q&A] [answered]
How do I go about creating multiple slots for authors in the blog addon for this? We'd like to allow multiple people to make posts to it as time goes on. It just isn't obvious looking at the config how we go about doing this for a project like ours....

**Comment by Mister-Hope** [comment, marked as answer]
We already descride **clearly enogh in docs**: https://vuepress-theme-hope.github.io/v2/config/frontmatter/info.html#author

So please tell me  how can I get raid of these kind of questions?

**Reply by kaosine** [reply]
I guess I should've dug deeper into that part (if I'm understanding your wording, since there is a bit of a language barrier, so I apologize if I misunderstand). It would help in my opinion if there was a bit about blog front matter on the section about that under getting started. As well, possibly not formatted as: 

```
type AuthorInfo = { name: string; url?: string };

type Author = string | string[] | AuthorInfo | AuthorInfo[];
```

That's not indicative of frontmatter in the way vuepress expects it, and that's the part that confuses me looking at it. Plus, I overthink things a lot and make them more difficult than they need to be...

**Reply by Mister-Hope** [reply]
That's typescript expressing. It's not easy to express a data structrure, but as you say, I would try to add some examples here.
